cfgov/jobmanager/models.py

```diff
@@ -102,8 +103,8 @@ def get_absolute_url(self):
 
     def save(self):
         if self.date_created == None:
```

Comment:
generally, i follow this rule: http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule

if i touch a file, and there's an improvement to be made, i want to leave the file in a better state than when i began making changes.